src/components/SearchBar/SearchBar.js

```diff
@@ -19,23 +16,22 @@ class SearchBar extends React.Component {
       sortBy: 'best_match'
 
     };
-  this.handleTermChange = this.handleTermChange.bind(this);
-  this.handleLocationChange = this.handleLocationChange.bind(this);
-  this.handleSearch = this.handleSearch.bind(this);
+    this.handleTermChange = this.handleTermChange.bind(this);
+    this.handleLocationChange = this.handleLocationChange.bind(this);
+    this.handleSearch = this.handleSearch.bind(this);
 
   }
   renderSortByOptions() {
     return Object.keys(sortByOptions).map(sortByOption => {
-      let sortByOptionsValue = sortByOptions[sortByOption];
+      const sortByOptionsValue = sortByOptions[sortByOption];
       return <li key={sortByOptionsValue} className={this.getSortByClass(sortByOptionsValue)} onClick={this.handleSortByChange.bind(this, sortByOptionsValue)}>{sortByOption}</li>;
     });
   }
   getSortByClass(sortByOption) {
-    if (this.state.sortBy === sortByOption)
-    {return 'active';}
-    else {
```

Comment:
Removed the else here - as when you hit a return it the code will jump out the function. The else is extra code that we can remove 👍